src/main/java/model/PunishmentType.java

```diff
@@ -9,15 +9,15 @@ public enum PunishmentType {
     EXPULSION,
     NONE;
 
-    public static PunishmentType calculateType(Map<AttendanceType, Integer> counts) {
-        int absenceCount = counts.get(AttendanceType.ABSENCE) + counts.get(AttendanceType.BE_LATE) / 3;
-        if (absenceCount > 5) {
+    public static PunishmentType find(Map<AttendanceType, Integer> attendanceTotal) {
+        Integer absenceTotal = attendanceTotal.get(AttendanceType.ABSENCE);
+        if (absenceTotal > 5) {
```

Comment:
```suggestion
        int absenceTotal = attendanceTotal.get(AttendanceType.ABSENCE);
        if (absenceTotal > 5) {
```

5, 3, 2는 뭘 의미하는지 상수로 선언하는 것은 어떨까요?

Reply:
아 상수 추출하면 더 용이할 것 같아요. 조언 감사합니다!